caddyhttp/httpserver/plugin.go

```diff
@@ -498,6 +498,7 @@ var directives = []string{
 	"grpc",      // github.com/pieterlouw/caddy-grpc
 	"gopkg",     // github.com/zikes/gopkg
 	"restic",    // github.com/restic/caddy
+	"nobots",    // github.com/Xumeiquer/nobots
```

Comment:
No, I believe a proper place should be at the top. Actually behind log. I am going to modify my commit.